rollup.config.js

```diff
@@ -0,0 +1,104 @@
+const fs = require('fs');
+const babel = require('rollup-plugin-babel');
+const readPkgUp = require('read-pkg-up');
+const resolve = require('rollup-plugin-node-resolve');
+const json = require('rollup-plugin-json');
+const commonjs = require('rollup-plugin-commonjs');
+const postcss = require('rollup-plugin-postcss');
+const peerDeps = require('rollup-plugin-peer-deps-external');
+const getBabelPreset = require('@commercetools-frontend/babel-preset-mc-app');
```

Comment:
Nitpick: move it after/before all the "rollup" requires, just to order the imports and have then more readable 😇